### DISCUSSION TITLE
Tuần quằng quằng

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#37] [General]
Gửi bài làm đã áp dụng kĩ thuật word lên đây hoặc code nha

**Comment by dangxuantruong10a13** [comment]
Cũng tới tuần này, tuần cuối ròu nhiệm vụ giao cho mn làm cũng ổn áp pket cho nên là mình sẽ theo kế hoạch ban đầu tuần này mình bắt đầu up file cuối mà mình làm đc lên code hoặc ở đây rồi tui xem có cần gì sửa nữa không r tui trang trí tổng hợp word gửi thầy nha

**Comment by dangxuantruong10a13** [comment]
Àk thì làm xong hết nhiệm vụ của mình rồi thì tuần cuối này có nv khác đổi gió nha, kkkkkkkkkk Cũng là làm word hay giúp tui trang trí hoi

**Reply by dangxuantruong10a13** [reply]
Tại bài thực hành word nên mọi người cần làm word hết như tui nói hồi tuần 2 á 💆💆💆💆💆👁️👄👁️
Mỗi bạn sẽ được cho bài word tổng hợp mấy bài kia xong từng bạn làm kĩ thuật thầy dặn. Vd như là kĩ thuật header ghi nhóm r footer hiện thị số trang thì cho cái này là Ân làm. Sau khi làm xong thì Ân gửi lên code hoặc vô đây r các bạn còn lại có nhiệm vụ còn lại cũng vậy. Cuối cùng là bỏ vô file word chính từ đó mà trang trí tkoi

**Comment by dangxuantruong10a13** [comment]
Ròi ai cũng gửi bài hết r giờ chia việc trc nèk

**Reply by TranBaoLam10C13** [reply]
Tui lm page 2 à

**Comment by dangxuantruong10a13** [comment]
Lâm làm trang 2 như yêu cầu của thầy

**Reply by TranBaoLam10C13** [reply]
Ai định dạng bài t sai hay j đó thì sửa lại cho đẹp nha

**Reply by dangxuantruong10a13** [reply]
là ai thấy định dạng bài ông sai thì sẽ sửa lại àk

**Comment by dangxuantruong10a13** [comment]
Hân làm phần trang 3 từ định dạng văn bản tới chia cấp độ
![Screenshot_2022-03-23-16-46-57-06_c37d74246d9c81aa0bb824b57eaf7062.jpg](https://user-images.githubusercontent.com/91663261/159671099-412db307-cc72-43af-8fa3-fe657a1a426d.jpg)

**Reply by NgHan12pn** [reply]
Tạo danh sách theo cấp độ là lấy dẫn chứng trong bài Bảo Lâm đk ??

**Reply by dangxuantruong10a13** [reply]
à không cái đó khoan đi, chỗ dẫn chứng của Lâm có gạch đầu dòng nhưng mà thoi đổi cái khác, cái khác ở đây thì chưa biết :v nên là làm việc khác đi

**Reply by NgHan12pn** [reply]
Okayy

**Reply by NgHan12pn** [reply]
Ủa mà sao lại phải đổi sang cái khác

**Reply by dangxuantruong10a13** [reply]
tại dẫn chứng của Lâm không có tạo thành list nên không xài được. Xài cũng được mà nhìn rác rác

**Reply by NgHan12pn** [reply]
Z đổi sang cái j thì báo cho tui bít để làm nha

**Reply by dangxuantruong10a13** [reply]
Giờ bà đánh máy ra mấy cái ý chính của mỗi đoạn trong mở bài, thân bài,kết bài. Ví dụ nha Thân bài cho lv1 xong xuống lv 2 là các ý như giải thích dẫn chứng,... Lv3 cho nội dung tổng của các đoạn đó

**Comment by dangxuantruong10a13** [comment]
Lan Anh từ chèn kí tự tới chèn hình ảnh trong phân trang 3 trở đi ![Screenshot_2022-03-23-16-46-57-06_c37d74246d9c81aa0bb824b57eaf7062.jpg](https://user-images.githubusercontent.com/91663261/159671935-987ee30f-4a02-4829-8c87-64ae4327cb26.jpg)

**Reply by 04lananh** [reply]
Là t chèn các kí tự r chỉnh ảnh hiệu ứng thui đk?

**Reply by dangxuantruong10a13** [reply]
Yes, kí tự đặc biệt thì xài symbol tool é. Chỉnh hiệu ứng hình ảnh tki làm sao vjp vjp là đc ;v

**Reply by 04lananh** [reply]
Ok đại ka🙆‍♀️🙆‍♀️🙆‍♀️

**Comment by dangxuantruong10a13** [comment]
Ân làm trang cuối như thầy yêu cầu

**Reply by dangxuantruong10a13** [reply]
![Screenshot_2022-03-23-16-46-57-06_c37d74246d9c81aa0bb824b57eaf7062.jpg](https://user-images.githubusercontent.com/91663261/159672162-a3a1c4d8-0e9e-43cd-9c51-d3eed3ba4db1.jpg)

**Reply by dangxuantruong10a13** [reply]
Ân làm 2 giai đoạn. Đầu tiên là làm phần mà đã tổng hợp chỉ có một số kĩ thuật (chưa có trang 2, các kĩ thuật mà Lan Anh với Hân sẽ làm). Insert chú thích tại chỗ áp dụng kĩ thuật. Phần sau là chú thích mấy kĩ thuật làm r ở trên. Nhớ ghi thông số

**Reply by dangxuantruong10a13** [reply]
[Tổng hợp cực chất.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8347505/T.ng.h.p.c.c.ch.t.docx)
rồi Ân bắc đầu làm việc nha. Sử dụng endnote trong reference để làm nha, xem cái cuối với trang áp cuối mà làm theo, tui có làm mẫu 2 chỗ ở trang áp cuối (Kết bài) và để ở sau chỗ mà áp dụng kĩ thuật rồi bấm vô endnote xong ghi kĩ thuật đó ra
![image](https://user-images.githubusercontent.com/91663261/160056280-b4cbca67-4521-42b1-ab07-1003f07c8bf0.png)

**Reply by TangVinhAn-PNc13** [reply]
yes sir. I fully understand the point you are making, thank you sir

**Reply by TangVinhAn-PNc13** [reply]
hope you have a great day

**Comment by TranBaoLam10C13** [comment]
nguồn: https://www.thanhuytphcm.vn/tin-tuc/nhung-tam-long-nhan-ai-giua-mua-dich-covid-19-1491880624
             https://tuoitre.vn/nguoi-hung-nguyen-ngoc-manh-lan-toa-nghia-cu-cao-dep-20210302102426804.htm

**Comment by TranBaoLam10C13** [comment]
![Capture](https://user-images.githubusercontent.com/91810620/159739270-2a55dabe-1663-4806-9ab3-9ff086f76608.PNG)
làm kiểu gì :(((

**Comment by TranBaoLam10C13** [comment]
vô xem thử có thiếu j ko để tui up lên code

**Reply by TranBaoLam10C13** [reply]
[trang 2 word.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8341341/trang.2.word.docx)
)

**Reply by dangxuantruong10a13** [reply]
Ổn r anh, mục lục ông xài table of contents đúng k hay sao. Theo tui nghĩ cái table of contents này mà đối với cái yêu cầu thầy nói (Trích thơ ra) thì 1 là viết giống ông chắc là ổn rồi. 2 là làm đúng chức năng cái này, là trong bài khi tới chỗ có câu nói hoặc là mở bài thân bài hoặc kết bài phải ghi ra ngay đầu là CÂU NÓI THỨ 1,2,3 hoặc MỞ BÀI, THÂN BÀI, KẾT BÀI để cho nó thành heading 1, xong mới cho thành ngữ, tục ngữ câu nói các thứ ở dưới

**Reply by TranBaoLam10C13** [reply]
z là ổn r à

**Reply by TranBaoLam10C13** [reply]
ông nhớ sửa lại số trang nha

**Reply by TranBaoLam10C13** [reply]
tui up lên code

**Reply by dangxuantruong10a13** [reply]
um anh

**Comment by dangxuantruong10a13** [comment]
@04lananh @NgHan12pn tạo issues riêng để trao đổi tại công việc 2 bạn đan xen nhau á nên là chia thời gian sao cho hợp lý nha. file tổng hợp ở code lấy file đó

**Reply by 04lananh** [reply]
Dạ ok anh👌

**Reply by dangxuantruong10a13** [reply]
oke cki cho tới hết thứ 7 gửi lại bài lên code hoặc ở đây nke

**Comment by NgHan12pn** [comment]
Aloo cho tui hỏi cái tạo danh sách theo cấp độ làm sao ạ

**Reply by dangxuantruong10a13** [reply]
![image](https://user-images.githubusercontent.com/91663261/160265935-8e26ab0b-d401-4bf0-ad23-d33965d8fbbe.png)
bôi đen chỗ muốn đổi để đổi lv nè

**Reply by NgHan12pn** [reply]
Chỗ muốn đổi là chỗ nào

**Comment by 04lananh** [comment]
Các đại ka cho em hỏi làm sao để lưu file mà không bị lỗi symbol á

**Reply by dangxuantruong10a13** [reply]
gửi file đây thử, chứ chưa nghe bao giờ :v

**Reply by 04lananh** [reply]
[Tổng-hợp-cực-chất.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8356787/T.ng-h.p-c.c-ch.t.docx)

**Reply by 04lananh** [reply]
đây nha

**Reply by dangxuantruong10a13** [reply]
lúc xài cái đó thì font bà xài là gì 
![image](https://user-images.githubusercontent.com/91663261/160266240-16fa6215-1f81-4578-9be4-dac47d36f1ff.png)

**Reply by dangxuantruong10a13** [reply]
mà ngoài cái symbol ra thì có mẫy chỗ bị lỗi chữ nhiều quá :))
![image](https://user-images.githubusercontent.com/91663261/160266320-7391fde7-33c3-4d0d-902b-1c17a837f712.png)
![image](https://user-images.githubusercontent.com/91663261/160266336-14309459-2bc7-404b-8b07-43d88405b009.png)
![image](https://user-images.githubusercontent.com/91663261/160266343-a27203f0-c02b-42c8-b733-39c8cef81105.png)

**Comment by 04lananh** [comment]
![Uploading 16483545719036296213421865908717.jpg…]()

**Reply by dangxuantruong10a13** [reply]
rồi oke @NgHan12pn tải font stars1 để xài symbol này nha

**Reply by dangxuantruong10a13** [reply]
mốt gửi thầy t embed font luôn nên không sao cả

**Reply by 04lananh** [reply]
Đại ka mãi đỉnh😍😍😍

**Reply by dangxuantruong10a13** [reply]
mà nè cái symbol ngôi sao đó nó cứ lạ lạ, nên là sau mỗi câu nói, tục ngữ thì chèn symbol gì đó phù hợp là được (Liên quan nội dung,...) hay cho cái symbol mặt trời :v không dính tới ai

**Reply by dangxuantruong10a13** [reply]
bỏ ngôi sao ra đi nhìn kì quá sad

**Reply by NgHan12pn** [reply]
Ê sao trong symbol bên t kh có font stars1

**Reply by dangxuantruong10a13** [reply]
nên t kêu tải é

**Reply by dangxuantruong10a13** [reply]
mà thôi được rồi bỏ con sao la đi rồi

**Reply by NgHan12pn** [reply]
máy tính bên t kh cho tải font đó về nè =(((

**Reply by dangxuantruong10a13** [reply]
ngon dữ, thoi được r

**Reply by 04lananh** [reply]
dạ cho em gửi lại bài
[Tổng-hợp-cực-chất.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8356823/T.ng-h.p-c.c-ch.t.docx)

**Reply by dangxuantruong10a13** [reply]
oki Hân xài mấy ảnh được định dạng trong bài này nha còn symbol để sau cũng được

**Comment by NgHan12pn** [comment]
tui đã làm xong và up bài của tui lên code r á mng

**Reply by NgHan12pn** [reply]
ai hóng thì vô xem nha

**Reply by NgHan12pn** [reply]
à ở phần mở đầu tui có thêm cover page vào để bài thêm sinh động với đẹp mắt hơn á

**Reply by dangxuantruong10a13** [reply]
ngon quá tốt nka kkkkk

**Reply by dangxuantruong10a13** [reply]
mà cover page thiệt ra là bắt buộc ớ thank iu kkkkkk

**Comment by dangxuantruong10a13** [comment]
okay giờ còn mỗi Ân thoi nka, giai đoạn 2 để t làm luôn cho nhanh giờ gửi phần giai đoạn 1 đi kkkkk

**Reply by TangVinhAn-PNc13** [reply]
ông ơi tôi làm xong r tui muốn gửi để ông check thì để ở đâu vậy

**Reply by dangxuantruong10a13** [reply]
okay gửi trong code nka ông kkkkk

**Reply by TangVinhAn-PNc13** [reply]
okiiii kkk

**Reply by dangxuantruong10a13** [reply]
![image](https://user-images.githubusercontent.com/91663261/160672406-469136af-d545-4d8e-b003-7ab673cb4010.png)
![image](https://user-images.githubusercontent.com/91663261/160672600-adaec82a-e96f-4875-8f1e-97d308948510.png)
làm như vầy nha anh, để con trỏ chuột ở đằng trước cái chữ đằng sau chữ bị drop cap. với lại bỏ định dạng cho mấy cái notes đi. Bài này còn thiếu nhiều quá. làm hết mấy cái định dạng luôn anh:v

**Reply by dangxuantruong10a13** [reply]
tại thầy cho nguyên trang cuối để làm luôn nên vậy là ít quá kkkkk

**Reply by TangVinhAn-PNc13** [reply]
hở T.T để em làm lại đại ka

**Reply by dangxuantruong10a13** [reply]
okay 19h mai làm xong nha kkkkkk còn đưa thầy xem trước xong mới nộp nữa

**Reply by dangxuantruong10a13** [reply]
à nhớ là note thì mình bỏ định dạng đi ra nha :v

https://user-images.githubusercontent.com/91663261/160675372-4c2a2712-ae32-4423-9535-af7b149d04b7.mp4

**Comment by TangVinhAn-PNc13** [comment]
Tớ đã up bài của mình lên code rồi nhé mấy cậu check giúp tớ ^^

**Reply by dangxuantruong10a13** [reply]
![image](https://user-images.githubusercontent.com/91663261/160679626-60c5f538-46e3-4094-b84c-ed08544c0879.png)

**Reply by dangxuantruong10a13** [reply]
oke anh mai em sửa nốt phần còn lại xong trang trí đủ kiểu r gửi thầy kkkk

**Comment by dangxuantruong10a13** [comment]
hello anh em check file này nha kkkkk sửa xong với trang trí hết rồi giờ chỉ gửi thầy xem trước 1 lần r gửi vô teams luôn kakakakakakakaka
[Bài thực hành word - Nhóm 9 - 10A13.docx](https://github.com/dangxuantruong10a13/nhom9_10a13/files/8377089/Bai.th.c.hanh.word.-.Nhom.9.-.10A13.docx)
k

**Reply by TranBaoLam10C13** [reply]
Ok bài đẹp r

**Reply by TangVinhAn-PNc13** [reply]
qua dep xung dang 11/10

**Reply by dangxuantruong10a13** [reply]
11/9 luôn

**Comment by dangxuantruong10a13** [comment]
okay vậy up teams nha kkkkkk

**Reply by TangVinhAn-PNc13** [reply]
okkk up i ông chắc mấy bợn ngủ khò khò gòi xD

**Reply by TranBaoLam10C13** [reply]
Up đi sợ j 🤨😄🥲😅😚😋😂😟😟😜😞😎😜
